Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundMainViewModel.swift

```diff
@@ -159,9 +166,16 @@ final class PlaygroundMainViewModel: ObservableObject {
                     case .completed(let session):
                         let accounts = session.accounts.data.filter { $0.last4 != nil }
                         let accountInfos = accounts.map { "\($0.institutionName) ....\($0.last4!)" }
+                        let sessionInfo =
+"""
+session_id=\(session.id)
+account_names=\(session.accounts.data.map({ $0.displayName ?? "N/A" }))
+account_ids=\(session.accounts.data.map({ $0.id }))
+"""
+
                         UIAlertController.showAlert(
                             title: "Success",
-                            message: "\(accountInfos)"
+                            message: "\(accountInfos)\n\n\(sessionInfo)"
```

Comment:
do you need 2 linebreaks here?

Reply:
It's purposefully there to separate stuff. It's debatable aesthetics! 

Note that everything in "FinancialConnections Example" is by-definition code that is a bit more 'loose' in its implementation as its quick code to achieve easier testing.